### ISSUE TITLE
Unable to find type [StringQuotingEmitter] when converting to YAML on linux

### ISSUE THREAD

**Issue by valters-tomsons** [cloudbase/powershell-yaml#71]
When using `ConvertTo-Yaml` from example throws the following error on linux (works on windows):

```
> ConvertTo-Yaml @{"hello"="world"; "anArray"=@(1,2,3); "nested"=@{"array"=@("this", "is", "an", "array")}}
Unable to find type [StringQuotingEmitter]
```

`ConvertFrom-Yaml` works as expected though.

**System Information:**
Linux 5.8
Powershell 7.0.3

**Comment by chscott** [comment]
Same problem for me on Windows.

**Comment by gabriel-samfira** [comment]
Hi folks!

I am really sorry for the late reply on this. This issue was fixed in #97 thanks to @Eggdale .
